desktop/release-notes.md

```diff
@@ -24,12 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.13.0
+2022-10-18
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added the new Dev Environments CLI `docker dev`.
```

Comment:
I wonder if we could say what the Dev Environments CLI can do for me / why I would want to use it? Especially since it's at the top of the New list.

Reply:
Good point, @djs55. How about this?

- Dev Environments now has a CLI, `docker dev`, so you can create, list, and run Dev Envs via command line. Now it's easier to integrate Dev Envs into custom scripts.

(CC @lorenrh)